packages/server/test/hooks.js

```diff
@@ -53,21 +55,22 @@ const initializeTestServer = async (app) => {
 
 exports.mochaHooks = {
   beforeAll: async () => {
+    console.log('running before all')
     await unlock()
     await knex.migrate.rollback()
     await knex.migrate.latest()
-    console.log('running before all')
+    await init()
```

Comment:
why do we need to initialize here, if we run the init in the beforeEachContext?

Reply:
IIRC it's because I don't use beforeNewContext() in my test module, because of performance issues - manually truncating a couple of relevant tables works quicker for me.